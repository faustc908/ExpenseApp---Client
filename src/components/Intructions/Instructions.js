import React from 'react';

// Component to give user instructions on tracker route

const Instructions = () => {
    <div className='instructions'>
        <p></p>
    </div>
}

export default Instructions;