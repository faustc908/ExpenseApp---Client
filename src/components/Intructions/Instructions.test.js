import React from 'react';
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import Instructions from './Instructions'
import ReactDOM from 'react-dom';

describe(`Instructions component`, () => {

    it('renders a Instructions by default', () => {
        const wrapper = shallow(<Instructions />)
        expect(toJson(wrapper)).toMatchSnapshot()
    })

    it('renders without crashing', () => {
        const div = document.createElement('div');
        ReactDOM.render(<Instructions />, div);
        ReactDOM.unmountComponentAtNode(div);
    });
})