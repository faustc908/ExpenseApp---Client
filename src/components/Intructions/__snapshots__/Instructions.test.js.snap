// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Instructions component renders a Instructions by default 1`] = `""`;
